RELEASE_NOTES.md

```diff
@@ -1,75 +1,22 @@
 # Changelog
 
-## [10.8.0] - 13.04.2023
+## [10.8.1] - 20.04.2023
```

Comment:
@stepandersen  don't worry about this changelog date & details
so far it's just to keep track of what's changed, we refine it before releasing, so it's easier to remember all the changes